contracts/clients/src/OperationResults.ts

```diff
@@ -117,28 +119,49 @@ const getError = (
 export const getOperationResults = (
   txnReceipt: ContractReceipt,
 ): OperationResult[] => {
-  if (!txnReceipt.events?.length) {
-    throw new Error(
-      `no events found in transaction ${txnReceipt.transactionHash}, are you sure it is a ContractReceipt from a VerificationGateway.processBundle transaction?`,
+  let walletOpProcessedEvents: Result[];
+
+  if (txnReceipt.events !== undefined) {
```

Comment:
Not a blocking comment, but is it worth adding a test for this new logic? e.g. `decodes WalletOperationProcessed logs`. At the moment, we only have a positive test case for a `ContractReceipt` with events, not with logs.

Reply:
Yeah good point. I found another little issue by doing this and fixed it too: https://github.com/web3well/bls-wallet/pull/597.